npg_qc_viewer/Changes

```diff
@@ -1,5 +1,6 @@
 Changes for NPG SeqQC
 
+ - Change default port for NPG-Tracking url
```

Comment:
This file was used when npg_qc_viewer was a stand-alone project. Then it was merged into npg_qc and we started to use the Changes file on the top level of the repo to track all changes. Could you move this line to the top-level file?